### ISSUE TITLE
Inherited values are not cleared when saving child page

### ISSUE THREAD

**Issue by InvisibleKind** [FluidTYPO3/flux#646]
Hi,

I've started testing inheritance mechanism and tried out it with relation field. For this purpose, I've enabled "clear" checkbox for this field.

When I tick this checkbox and save a record - everything is ok, and value is inherited from parent, but checkbox doesn't appear as "checked", so saving page again without any changes (or with some non-related changes, like title, alias, etc.) will reset clearing state and no inheritance will happen again.

Also it is not obvious for an editor, is the field inherited from parent or not by looking on a FlexForm, because there is no visual indicator for that.

**Comment by NamelessCoder** [comment]
> 5. Those children, which get their titles (or any other fields, except 'footerPid') edited before, will have footerPid = 2.

This indicates a problem in the inheritance: the pages which were edited should not have any value stored as `footerPid` - this value should have been removed when saving the record, because Flux should have detected that an identical value will be inherited. If this was working you would not be experiencing the problem at all.

I've changed the title of the issue accordingly.
